engine/apps/phone_notifications/models/phone_call.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+
+
+# Duplicate to avoid circular import
+class TwilioCallStatuses:
```

Comment:
This will be removed after migration, right?

Reply:
Yes, I'm keeping it to be able to receive Gather (digits from phone call) for calls which were made shortly before migration.